test/unit/transform/eia861.py

```diff
@@ -0,0 +1,106 @@
+"""Unit tests for the pudl.transform.eia861 module."""
+
+from io import StringIO
+
+import pandas as pd
+import pytest
+
+from pudl.metadata.fields import apply_pudl_dtypes
+from pudl.transform import eia861
+
+# Test that rows with the same primary key are combined correctly
+actual_1 = pd.read_csv(
+    StringIO(
+        """report_date,utility_id_eia,state,ba_code,value
+2019-01-01,88888,TX,ERCOT,100
+2019-01-01,88888,TX,ERCOT,300
+2019-01-01,88888,pd.NA,ERCOT,800
```

Comment:
Sorry for the drive-by comment but is the literal string "pd.NA" what we want here? Or should it just be a blank entry, which will be turned into an actual pandas null value by virtue of the `apply_pudl_dtypes`?